imod_coupler/drivers/metamod/config.py

```diff
@@ -40,6 +40,7 @@ class Coupling(BaseModel):
     mf6_msw_sprinkling_map: Optional[
         FilePath
     ] = None  # the pach to the sprinkling map file
```

Comment:
```suggestion
    ] = None  # the path to the sprinkling map file
```

Not your typo, but now that I see it 😉